codecs/hqx/build.sh

```diff
@@ -9,6 +9,9 @@ echo "============================================="
   rustup run nightly \
     wasm-pack build --target no-modules
   wasm-strip pkg/squooshhqx_bg.wasm
+  echo "Optimising WASM so it doesn't break Chrome (this takes like 10-15mins. get a cup of tea)"
```

Comment:
nit: `/s/WASM/Wasm/`

Reply:
Fixed in 2f834db22499af1b7ff1e7c050a52b9a0d0e2633

Reply:
that’s still wrong lolol

it’s supposed to be “Wasm” with (only) the “W” capitalized

Reply:
https://github.com/GoogleChromeLabs/squoosh/pull/630